packages/web/src/components/nav/desktop/NavAudio.tsx

```diff
@@ -87,22 +117,13 @@ const RenderNavAudio = () => {
           leave={{ opacity: 0 }}
           config={{ duration: 100 }}
         >
-          {(item) => (props) =>
-            item !== 'none' && (
-              <animated.span
-                style={props}
-                className={cn(styles.actionBubble, styles.interactive, {
-                  [styles.claimRewards]: item === 'claim'
-                })}
+          {(bubbleType) => (bubbleStyle) =>
```

Comment:
Hm not familiar with this pattern, does omitting line 120 and just doing `style={styles.bubbleStyle}` not work?

Reply:
`style` is passed in by `<Transition />`. It's how react-spring does the CSS to animate the element, I believe.